packages/reports/src/Stores/index.ts

```diff
@@ -4,26 +4,47 @@ import StatementStore from './Modules/Statement/statement-store';
 import type { TStores } from '@deriv/stores';
 
 export type TRootStore = TStores & {
+    active_symbols: Record<string, any>;
+    chart_barrier_store: Record<string, any>;
+    contract_replay: Record<string, any>;
+    gtm: Record<string, any>;
     modules: {
         profit_table: ProfitTableStore;
         statement: StatementStore;
     };
+    notifications: Record<string, any>;
+    pushwoosh: Record<string, any>;
+    rudderstack: Record<string, any>;
 };
 
 export default class RootStore {
-    constructor(core_store: TStores) {
+    public active_symbols: Record<string, any>;
+    public chart_barrier_store: Record<string, any>;
+    public client: Record<string, any>;
+    public common: Record<string, any>;
+    public contract_replay: Record<string, any>;
+    public contract_trade: Record<string, any>;
+    public gtm: Record<string, any>;
+    public modules: Record<string, any>;
+    public notifications: Record<string, any>;
+    public portfolio: Record<string, any>;
+    public pushwoosh: Record<string, any>;
+    public rudderstack: Record<string, any>;
+    public ui: Record<string, any>;
```

Comment:
@akmal-deriv Why setting `Record<string, any>` for everything? 🤔

Reply:
This follows how it's done in other packages, you can leave any suggestions if you have

Reply:
Annotating everything as `any` means no TS check at all, We should type these shared stores in `packages/stores/types.ts`, Some of the stores are already partially typed there like `TUiStore` so we can improve the types there and reuse it everywhere.